src/Storage/Storage.Management/StorageAccount/NewAzureStorageBlobInventoryPolicyRule.cs

```diff
@@ -87,15 +92,47 @@ public class NewAzureStorageBlobInventoryPolicyRuleCommand : StorageAccountBaseC
             BlobInventoryPolicyBlobSchemaField.LastAccessTime,
             BlobInventoryPolicyBlobSchemaField.AccessTierInferred,
             BlobInventoryPolicyBlobSchemaField.Tags,
+            BlobInventoryPolicyBlobSchemaField.Etag,
+            BlobInventoryPolicyBlobSchemaField.ContentType,
+            BlobInventoryPolicyBlobSchemaField.ContentEncoding,
+            BlobInventoryPolicyBlobSchemaField.ContentLanguage,
+            BlobInventoryPolicyBlobSchemaField.ContentCRC64,
+            BlobInventoryPolicyBlobSchemaField.CacheControl,
+            BlobInventoryPolicyBlobSchemaField.ContentDisposition,
+            BlobInventoryPolicyBlobSchemaField.LeaseStatus,
+            BlobInventoryPolicyBlobSchemaField.LeaseState,
+            BlobInventoryPolicyBlobSchemaField.LeaseDuration,
+            BlobInventoryPolicyBlobSchemaField.ServerEncrypted,
+            BlobInventoryPolicyBlobSchemaField.Deleted,
+            BlobInventoryPolicyBlobSchemaField.RemainingRetentionDays,
+            BlobInventoryPolicyBlobSchemaField.ImmutabilityPolicyUntilDate,
+            BlobInventoryPolicyBlobSchemaField.ImmutabilityPolicyMode,
+            BlobInventoryPolicyBlobSchemaField.LegalHold,
+            BlobInventoryPolicyBlobSchemaField.CopyId,
+            BlobInventoryPolicyBlobSchemaField.CopyStatus,
+            BlobInventoryPolicyBlobSchemaField.CopySource,
+            BlobInventoryPolicyBlobSchemaField.CopyProgress,
+            BlobInventoryPolicyBlobSchemaField.CopyCompletionTime,
+            BlobInventoryPolicyBlobSchemaField.CopyStatusDescription,
+            BlobInventoryPolicyBlobSchemaField.CustomerProvidedKeySha256,
+            BlobInventoryPolicyBlobSchemaField.RehydratePriority,
+            BlobInventoryPolicyBlobSchemaField.ArchiveStatus,
+            BlobInventoryPolicyBlobSchemaField.xmsblobsequencenumber,
+            BlobInventoryPolicyBlobSchemaField.EncryptionScope,
+            BlobInventoryPolicyBlobSchemaField.IncrementalCopy,
+            BlobInventoryPolicyBlobSchemaField.DeletionId,
+            BlobInventoryPolicyBlobSchemaField.DeletedTime,
+            BlobInventoryPolicyBlobSchemaField.TagCount,
```

Comment:
Please make sure the blob/container schemafields are aligned with the swagger description in https://github.com/Azure/azure-rest-api-specs/blob/894645e485d2f01e83999480d440e1bace3fec9b/specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json#L5258

Reply:
Done.